### PR TITLE
Document battle background and terrain

### PR DESCRIPTION
**Pull request by cbt6**

### CODE REVIEW

**Review by tgsm — DISMISSED** [review]

File: src/battle/battle_setup.c [hunk 1]
```diff
@@ -466,29 +466,29 @@ void sub_02052444(BattleSetup* setup, FieldSystem* fieldSystem) {
 }
 
 static const u32 _020FC4C0[] = {
```
Comment (red031000) [inline src/battle/battle_setup.c:468]: any chance to name this var, smth like BattleBgTerrainMap?

**Review by red031000 — CHANGES_REQUESTED** [review]
in addition to this, I'd also say make sure that vars/struct members that expect these enums are given the proper type, since right now it seems a mix between the enum and u32 or smth

File: src/battle/battle_setup.c (hunk 1 above)
Comment (cbt6) [reply src/battle/battle_setup.c]: Let's keep the renaming of variables and functions to its own PR.

Comment (red031000) [reply src/battle/battle_setup.c]: I mean, it's not massive but if you want to do it separately then sure

**Review by red031000 — APPROVED** [review]

**Pull request merged**